controllers.js

```diff
@@ -43,7 +44,18 @@ exports.getArticleById = (req, res, next) => {
 exports.getCommentsByArticleId = (req, res, next) => {
   selectCommentsByArticleId(req)
     .then((comments) => {
-        res.send({ comments });
+      res.send({ comments });
+    })
+    .catch((err) => {
```

Comment:
try to avoid adding changes to code that's already been pushed, it can be quite confusing to review and means that the history of this ticket and branch are not exactly what happened